deploy

```diff
@@ -23,67 +21,15 @@ import yaml
 import argparse
 import subprocess
 
-# the code supports multiple prod pipelines for easing migrations
-PROD_CLUSTERS = {
-    "https://api.ocp.fedoraproject.org:6443": "fedora-coreos-pipeline"
-}
-
-
 def main():
     args = parse_args()
-
-    if targeting_official_namespace(args) and not args.official:
-        eprint("Refusing to create developer resource in official namespace.")
-        eprint("Use --official to create official resources.")
-        return 1
```

Comment:
Can we keep this check but just delete the "developer" word in there? It's a simple fail-safe to ensure that you're not deploying against the official namespace by mistake.

Reply:
I was kind of trying to get away from hardcoding the URL in various places but I think maybe it's OK to keep this.